interval/lift.go

```diff
@@ -43,7 +43,8 @@ func ReadToLiftChan(inputFile string, send chan<- Lift) {
 	case ".vcf":
 		receive, _ := vcf.GoReadToChan(inputFile)
 		for val := range receive {
-			send <- val
+			curr := val
```

Comment:
I don't see a bug here, but I am curious why we don't send val directly and instead make a copy of it into curr and then send curr?